footprints/main/views.py

```diff
@@ -317,8 +317,45 @@ def get_rows(self):
                    'Imprint Actor and Role', 'Imprint BHB Number',
                    'Imprint OCLC Number', 'Evidence Type', 'Evidence Location',
                    'Evidence Call Number', 'Evidence Details']
+        for r in Role.objects.for_footprint():
+            role = 'Footprint Role ' + unicode(r.name)\
```

Comment:
The other places in this commit that are calling `unicode()` should be updated to use `smart_text` as well